.github/workflows/gotests.yml

```
@@ -22,13 +22,13 @@ on:
 jobs:
   gotests:
     runs-on: ubuntu-latest
-    name: Tests for Go 1.24
+    name: Tests for Go 1.25
     steps:
       - uses: actions/checkout@v4
       - name: Setup Go
         uses: actions/setup-go@v4
```

actionlint (1.7.10)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)